.github/ISSUE_TEMPLATE/bug.yaml

```
@@ -16,6 +16,14 @@ body:
       options:
         - label: This is not a support question.
           required: true
+  - type: checkboxes
+    id: labels
+    attributes:
+      label: Have you labeled this issue correctly?
+      description: Please label this issue correctly, specially with one of the **Tx** ones. Check <https://paritytech.github.io/labels/doc_substrate.html> for more info.
+      options:
+        - label: I have labeled this issue correctly.
+          required: true
```

but isn't it impossible for external contributors to label? answering is a little dilemma
minor suggestion
```suggestion
      label: Have you labeled this issue correctly? (or N/A)
      description: If applicable please label this issue correctly, specially with one of the **Tx** ones. Check <https://paritytech.github.io/labels/doc_substrate.html> for more info.
      options:
        - label: I have labeled this issue correctly or N/A.
          required: true
```

---

Yes I think so as well.  
@juangirini wanted to confirm this with an external account. If so, he wanted to close this MR.

---

I have double checked it and indeed it is not possible for external contributors to label issues/prs. But still existing documentation mentions labeling, see the [Contributing doc](https://github.com/paritytech/substrate/blob/e5763b825fa41e05a49094bef3c67300999a52a5/docs/CONTRIBUTING.adoc) or the [PR template](https://github.com/paritytech/substrate/blob/e5763b825fa41e05a49094bef3c67300999a52a5/docs/PULL_REQUEST_TEMPLATE.md).

I am happy to close this PR, if I do so we might want to consider whether we want to mention the labels on those other documents and if we want to make the distinction between Parity and external contributors.

If we merge, surely Guillaume's changes make a lot of sense.

I don't have a strong opinion on whether to close this or merge it. But in either case we should be consistent across different documentation. What do you think? cc @kianenigma @paritytech/docs-audit

---

All in all, I see two tasks to be done:

1. For internal people who have the permission, they should know the latest rules about labeling. We could add a bot that would warn them if they have opened an issue, but not labeled it. All of this requires the bot to check if the user of choice is part of our org or not.
2. For external people, we just need a way for any issue created by them to marked with a label called `needs-verification` which is mainly a signal for @juangirini or someone else to triage it. I think `Unconfirmed` is doin that now. 

If this PR helps with any of these, then it is good, otherwise it can be closed. 

From what I can see, it can still be useful, but has to specify a different template for those within parity, and without.

---

Indeed 2 is already in place with the `j2-unconfirmed` label